returns/fix_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"flag"
 	"testing"
 
-	_ "golang.org/x/tools/go/gcimporter"
+	_ "go/importer"
```

Comment:
Really minor, but it might be nicer to group `"go/importer"` together with the rest of standard library imports above.